### ISSUE TITLE
Add conda-forge feedstock

### ISSUE THREAD

**Issue by jbusecke** [jbusecke/xmovie#37]
It would be nice to have a conda-forge feedstock, especially if that could include ffmpeg as dependency. 

Not having ffmpeg on the system has caused some trouble in #29.

**Comment by jbusecke** [comment]
Started the process in https://github.com/conda-forge/staged-recipes/pull/14543

**Comment by tomchor** [comment]
It seems like it was already merged into conda, no?

**Comment by jbusecke** [comment]
Yes! Closing this now.

**Comment by tomchor** [comment]
I noticed that the version available at conda is still 0.1 though. Are you planning on tagging a 0.2 release?
I ask in order to decide if I upgrade via conda now or wait a bit more until the new version gets released (to avoid "version confusion" in my dependency records :) )

**Comment by jbusecke** [comment]
Yes I wanted to get the 'old version' out first, in case there were any hickups. I think I can release `0.2.0` after #41 is resolved.

**Comment by jbusecke** [comment]
Here it is: https://github.com/jbusecke/xmovie/releases/tag/v0.2.0

**Comment by jbusecke** [comment]
I just released a new version, but the publishing action failed. Ill have to deal with this tomorrow/Thu. Sorry for the inconvenience

**Comment by tomchor** [comment]
Awesome! No need for such rush on my end, though. As long as it's done this week it's okay for me to wait :)
